exp-player/addon/styles/components/isp-radio-group.scss

```diff
@@ -24,6 +24,11 @@
   margin-top: -15px;
 }
 
-.label-spacing {
-  max-width: 6%;
+.measure-four {
+  width: 6%;
```

Comment:
I tried re-using classes when possible (e.g. The `format-label` class is used for questions 2 & 3) but the formatting looked off in some cases